### ISSUE TITLE
Feet distance reward bug

### ISSUE THREAD

**Issue by Nike353** [ziyanx02/Genesis-backflip#4]
Hi. In the reward that maintains a constant feet stance width, the stance width should be 0.3* torch.ones instead of 0.3* torch.zeros
